app/src/main/kotlin/com/github/funkyg/funkytunes/network/PirateBayAdapter.kt

```diff
@@ -24,10 +24,10 @@ import kotlin.collections.ArrayList
 class PirateBayAdapter(context: Context) {
 
     private val Tag = "PirateBayAdapter"
-    private val DOMAIN = "https://theproxypirate.pw"
+	private val DOMAINS: Array<String> = arrayOf("https://pirateproxy.cc", "https://pirateportal.xyz", "https://thepiratebay.org")
```

Comment:
You dont need to specify the type of DOMAINS, kotlin does that automatically. Also your intendation looks wrong.